### ISSUE TITLE
Decimal separator according to locale

### ISSUE THREAD

**Issue by budda85** [OpenOlitor/OpenOlitor#141]
For tracking I open another ticket here, even if I have already written in December with Alwin.
Maybe someone else has a suitable solution.

If you want to create invoices and then offset the amounts with VAT via Libreoffice, it unfortunately no longer works. [2021 it worked once.](https://github.com/OpenOlitor/OpenOlitor/issues/53)
This is due to the decimal separator. If you want to calculate the values by formulas in Libreoffice, wrong values come out, because OO outputs a point as decimal separator.

![Rechnung](https://user-images.githubusercontent.com/20152200/237030020-037ecbf8-e0d6-41c3-996d-565cd7aa3061.png)

In Germany we use the comma as decimal separator. 
All my attempts to set Libreoffice to English or Swiss to work around the problem have failed.
Alwin had already reproduced the problem with himself if I understood it correctly.
The problem is, as far as I have found out, with the server container. Here it doesn't help if I set the appropriate "locale". 
![locale-docker-compose](https://user-images.githubusercontent.com/20152200/237030105-7f9042ca-378d-451a-9cbf-b73c348de020.png)

My Linux server and e.g. the pdf-tools container convert the dot to a comma. The server container unfortunately does not
![Unterschiede-container](https://user-images.githubusercontent.com/20152200/237030322-7ce8b14b-91d4-439c-b8d9-39447b033f02.png)

However, the server container is missing the matching locale as I see it.
![fehlende-locale](https://user-images.githubusercontent.com/20152200/237030456-7bc17d5a-420e-4a1d-8669-0637da157ac4.png)

Here it would be nice if there is a solution so that you can use the invoicing and I don't have to go the detour via a mail merge.

**Comment by budda85** [comment]
Is there anything new here?
We have to create invoices for this year.

**Comment by mcmpp** [comment]
Hi @budda85 , 

I am having a look at this right now. I would like to have more information about it. 
* Can you explain me more about your workflow: it seems that your template adds the TVA. Every invoice has the same percentage? 
* could I have the template that you are using for the invoice?
* Is this issue only reflected while creating the invoices?

**Comment by budda85** [comment]
Hi @mcmpp 

- You had to translate TVA for me, that I don't know. :-) We have two percentages. Once 7% and once 19%. But the problem occurs with both values.
- The template which I used is [this one here](https://openolitor.org/mwst-betrag-in-mitgliederrechnung-ausweisen/), I have also tried a new one. Alwin has also created a template which also does not work. See this file: [Rechnungsvorlage-neu_Alwin2.odt](https://github.com/OpenOlitor/OpenOlitor/files/12468824/Rechnungsvorlage-neu_Alwin2.odt)
- Elsewhere, I have not encountered the problem until now. However, we used the tax calculation only in the invoices.

**Comment by budda85** [comment]
Hey @mcmpp is that for me? :-D
https://github.com/OpenOlitor/openolitor-server/releases/tag/2.6.23_test

**Comment by mcmpp** [comment]
Yes, it was but I am not sure this will change anything... you can try and let me know .
